src/middleware/router.js

```diff
@@ -432,8 +430,13 @@ function sendHttpRequest (ctx, route, options) {
     const routeReq = method.request(options, (routeRes) => {
       response.status = routeRes.statusCode
       response.headers = routeRes.headers
-      response.body = []
+      response.body = new Readable()
+      response.body._read = () => {}
```

Comment:
What does this `_read ` property do? And why do we need to attach an empty function to it?